cypress/integration/Test_Phonebook/test_register.js

```diff
@@ -0,0 +1,13 @@
+describe('Login Test', function(){
```

Comment:
Use the proper name for this test suite.

Reply:
done